plasma_framework/test/src/vaults/Erc20Vault.test.js

```diff
@@ -127,6 +130,26 @@ contract('Erc20Vault', (accounts) => {
                 'Must have only one output',
             );
         });
+
+        it('deposit verifier waits a period of time before takes effect', async () => {
+            const depositValue = DEPOSIT_VALUE / 2;
+            const deposit = Testlang.deposit(depositValue, alice, this.erc20.address);
+            await this.erc20.approve(this.erc20Vault.address, depositValue, { from: alice });
+            expect(this.currentDepositVerifier).to.not.equal(null);
```

Comment:
are you checking the value from the contract or from the js?

Reply:
I think that you have just set it in the before code, so you do not have to explicitly check it 😉

Reply:
Oh this `expect` is stupid - :+1: